build/charts/antrea/crds/bgppolicy.yaml

```diff
@@ -0,0 +1,125 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: bgppolicies.crd.antrea.io
+spec:
+  group: crd.antrea.io
+  versions:
+    - name: v1alpha1
+      served: true
+      storage: true
+      schema:
+        openAPIV3Schema:
+          type: object
+          required:
+            - spec
+          properties:
+            spec:
+              type: object
+              required:
+                - nodeSelector
```

Comment:
I think `localASN` should be added to the required properties after the recent changes?

Reply:
I forgot adding this. Will add it. Thanks for pointing out it.